chapters/data-analysis.tex

```diff
@@ -1,182 +1,237 @@
+Whenever possible, you should track have each of them with the code that created it.
+If you are writing code in Git or GitHub,
 you can output plain text files such as \texttt{.tex} tables
 and metadata saved in \texttt{.txt} or \texttt{.csv} to that directory.
 Binary files that compile the tables,
 as well as the complete data sets, on the other hand,
-should be stored in your team's shared folder. 
+should be stored in your team's shared folder.
 Whenever data cleaning or data construction codes are edited,
 use the master script to run all the code for your project.
-Git will highlight the changes that were in data sets and results that they entail. 
+Git will highlight the changes that were in data sets and results that they entail.
 
 %------------------------------------------------
 
-\section{Data cleaning}
+\section{De-identifying research data}
+
+The starting point for all tasks described in this chapter is the raw data
+which should contain only materials that are received directly from the field.
+The raw data will invariably come in a host of file formats and these files
+should be retained in the raw data folder \textit{exactly as they were 
+received}. Be mindful of how and where they are stored as they can not be
+re-created and nearly always contain confidential data such as
+personally-identifying information\index{personally-identifying information}.
+As described in the previous chapter, confidential data must always be
+encrypted\sidenote{\url{https://dimewiki.worldbank.org/wiki/Encryption}} and be
+properly backed up since every other data file you will use is created from the
+raw data. The only data sets that can not be re-created are the raw data 
+themselves.
+
+The raw data files should never be edited directly. This is true even in the
+rare case when the raw data cannot be opened due to, for example, incorrect
+encoding where non-English character is causing rows or columns to break at the
+wrong place when the data is imported. In this scenario, you should create a
+copy of the raw data where you manually remove the special characters and
+securely back up \textit{both} the broken and the fixed copy of the raw data.
+You will only keep working from the fixed copy, but you keep both copies in
+case you later realize that the manual fix was done incorrectly.
 
-Data cleaning is the first stage of transforming the data you received from the field into data that you can analyze.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Cleaning}}
-The cleaning process involves (1) making the data set easily usable and understandable, 
+Loading encrypted data frequently can be disruptive to the workflow.
+To facilitate the handling of the data, remove any personally identifiable information from the data set.
+This will create a de-identified data set, that can be saved in a non-encrypted folder.
+De-identification,\sidenote{\url{https://dimewiki.worldbank.org/wiki/De-identification}}
```

Comment:
```suggestion
De-identification\sidenote{\url{https://dimewiki.worldbank.org/wiki/De-identification}},
```
the side note explains the term so set the number directly to the word

Reply:
This is a general formatting question. There are many places where this is inconsistently done. I personally think the note numbering in the text should come _after_ sentence stops, commas, and colons; and _before_ parentheses or quotation marks.

Reply:
I agree!

Reply:
I think it should come after if it references the clause, but immediately on the word if it reference a word or a terminology. This case is weird as it is a one word clause, but I think it this case is the second case. But I seem to be outvoted here, which is fine.